test/EnsembleKalmanProcess/runtests.jl

```diff
@@ -79,6 +79,123 @@ inv_problems = [inv_problems..., nl_inv_problems...]
 end
 
 
+@testset "Accelerators" begin
+    # Get an inverse problem
+    y_obs, G, Γy, _ = inv_problems[end - 1] # additive noise inv problem
+    rng = Random.MersenneTwister(rng_seed)
+    initial_ensemble = EKP.construct_initial_ensemble(rng, prior, N_ens)
+
+    # build accelerated and non-accelerated processes
+    ekiobj = EKP.EnsembleKalmanProcess(
+        initial_ensemble,
+        y_obs,
+        Γy,
+        Inversion(),
+        scheduler = DefaultScheduler(),
+        accelerator = NesterovAccelerator(),
+    )
+    eksobj = EKP.EnsembleKalmanProcess(
+        initial_ensemble,
+        y_obs,
+        Γy,
+        Sampler(prior),
+        scheduler = DefaultScheduler(),
+        accelerator = NesterovAccelerator(),
+    )
+    ekiobj_noacc = EKP.EnsembleKalmanProcess(initial_ensemble, y_obs, Γy, Inversion(), scheduler = DefaultScheduler())
+    eksobj_noacc =
+        EKP.EnsembleKalmanProcess(initial_ensemble, y_obs, Γy, Sampler(prior), scheduler = DefaultScheduler())
+    ekiobj_noacc_specified = EKP.EnsembleKalmanProcess(
+        initial_ensemble,
+        y_obs,
+        Γy,
+        Inversion(),
+        scheduler = DefaultScheduler(),
+        accelerator = DefaultAccelerator(),
+    )
+    eksobj_noacc_specified = EKP.EnsembleKalmanProcess(
+        initial_ensemble,
+        y_obs,
+        Γy,
+        Sampler(prior),
+        scheduler = DefaultScheduler(),
```

Comment:
I would remove all the scheduler = DefaultScheduler() comments, and use the default for each method (this will remove the EKS warnings)